i-slide.js

```diff
@@ -351,16 +375,11 @@ class ISlide extends HTMLElement {
 
       }
     } catch (err) {
-      console.error(err);
+      console.error(err.toString());
```

Comment:
doesn't that lose us visibility on the stack trace (at least when running in non-headless browser)?

Reply:
It does but many tests trigger errors on purpose and the former form yields an opaque non-human friendly structure in the console output when running the test suite on the command line. A workaround that seems to give the best of both worlds is to do `console.error(err.toString(), err);`. This gives a meaningful message when running the test suite on the command line, gives access to the stack trace in debug mode in the dev console, and only pollutes the command-line output with a `JSHandle@error` string at the end of the error line.

Reply:
```suggestion
      console.error(err.toString(), err);
```